fontforgeexe/cvpalettes.c

```diff
@@ -76,15 +76,16 @@ struct l2 {
     BDFChar **layers;     /* layer thumbnail previews              */
     int sb_start;         /* x pixel position of the scrollbar     */
     int column_width;     /* width of various indicator columns    */
+    int header_height;    /* height of the header in pixels before the first layer */
```

Comment:
This is even done by the normal layers palette, although for some reason it's declared outside the struct as `layer_header_height`.